### DISCUSSION TITLE
DEV_SSR Flag: Can't disabled

### DISCUSSION THREAD

**Discussion by muhrusdi** [gatsbyjs/gatsby#29644] [Help]
Wheter this feature can't be disabled?

![Capture](https://user-images.githubusercontent.com/9457288/108622824-5a025f80-7476-11eb-98e0-0a3513823c2c.PNG)

**Comment by herecydev** [comment]
You should be able to disable it as follows:

```js
//gatsby-config.js

module.exports = {
  flags: {
    DEV_SSR: false
  }
}

**Reply by muhrusdi** [reply]
i tried several times but cant disabled, it fixed now, thanks
